vllm-tensorizer/Dockerfile

```diff
@@ -47,17 +47,38 @@ RUN git clone --filter=blob:none --depth 1 --no-single-branch --no-checkout \
     git submodule update --init --recursive --jobs 8 \
       --depth 1 --filter=blob:none
 
+FROM alpine/git:2.36.3 as vllm-flash-attn-downloader
+WORKDIR /git
+ARG VLLM_FLASH_ATTN_VERSION
+RUN git clone --filter=blob:none --depth 1 --no-single-branch --no-checkout \
+      https://github.com/vllm-project/flash-attention.git && \
+    cd flash-attention && \
+    git checkout "v${VLLM_FLASH_ATTN_VERSION}" && \
```

Comment:
I imagine it's safe to assume the branch names won't ever conflict here and lead to ambiguity.